### PR TITLE
Shopware 6

### PR DESCRIPTION
**Pull request by dairbuirabass**
This PR will not be merged, but kept as a separate branch for Shopware 6 docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,5 +32,5 @@ Open Software License \("OSL"\) v3.0
```
Comment (dairbuirabass) [inline README.md]: @gary-nosto which license is Overdose plugin published under?

File: installing.md [hunk 2]
```diff
@@ -1,26 +1,8 @@
...
-
-After this, the plugin can be activated in the Shopware Plugin Manager.
-
-## Repository
-
-For development purposes the plugin can be installed directly from the GitHub repository by cloning the project into the `/engine/Shopware/Plugins/Local/Frontend/NostoTagging` directory of the Shopware installation. For the plugin to work, it's dependencies need to be installed. For this we recommend using [composer](https://getcomposer.org/), which is a dependency manager for PHP. By executing composer install in the plugins root folder, the dependencies will be automatically fetched and installed in a vendor folder relative to the plugin root directory.
-
-After this, the plugin can be activated in the Shopware Plugin Manager.
+- The plugin can also be installed as a "local" plugin by uploading the plugin archive file manually in the Shopware Administration. After this, the plugin can be activated.
+- With Composer
```
Comment (gary-nosto) [inline installing.md]: The plugin should only be installed through the Shopware store https://store.shopware.com/en/overd73641583843f/nosto-personalization-for-shopware-6.html

File: README.md (hunk 1 above)
Comment (gary-nosto) [reply README.md]: I just asked Overdose about this.

File: installing.md (hunk 2 above)
Comment (dairbuirabass) [reply installing.md]: Updated.

File: README.md (hunk 1 above)
Comment (dairbuirabass) [reply README.md]: Removed license.

**Review by gary-nosto — APPROVED** [review]

**Pull request closed without merging**